contrib/travis.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -ex
```

Comment:
nit: it may be easier to spot for the reader to write
```suggestion
#!/bin/sh

set -e
set -x
```